functions/src/etl/FirestoreAuditor.ts

```diff
@@ -0,0 +1,86 @@
+import equal from 'deep-equal';
```

Comment:
Could we use the `isEqual()` function from Lodash? I think it does the same thing and then we don't need an additional library...

Reply:
True, much better